conf/app.routes

```diff
@@ -0,0 +1 @@
+GET        /admin/metrics                                           com.kenshoo.play.metrics.MetricsController.metrics
```

Comment:
shouldn't this stay in `prod.routes` - if it's here then the url will be expected to be prefixed by `/cgt-property-disposals-stubs` now when it wasn't before

Reply:
Earlier I had some unused import issues with this file. I just moved this to app.routes to fix those issues. Let me test whether its required or not with new configs

Reply:
If I did this change, unfortunately Im getting these error:
`[error] /Users/ram/Applications/hmrc/CGTProjects/cgt-property-disposals-stubs/conf/app.routes: Unused import
[error] /Users/ram/Applications/hmrc/CGTProjects/cgt-property-disposals-stubs/conf/app.routes: private val defaultPrefix in class Routes is never used`

Is this app.routes really required? Its just empty

Reply:
it isn't required for now, but may be for the future. Let's delete it for now if it's not being used then and add it back in if we need it in the future